src/control/server/storage/bdev/backend.go

```diff
@@ -149,6 +156,21 @@ func (b *spdkBackend) Init(shmID ...int) error {
 	return nil
 }
 
+// EnableVmd turns on VMD device awareness.
+func (b *spdkBackend) EnableVmd() error {
+	if b.binding.vmdEnabled {
+		return errors.New("EnableVmd(): VMD already enabled")
+	}
```

Comment:
Maybe I'm missing something, is there some kind of consequence that could come from setting it twice? I'd be inclined to return nothing from this function